test/data/base/04-op-in-define.yaml

```diff
@@ -0,0 +1,12 @@
+otk.version: "1"
+
+otk.define.name:
+  foo:
+    otk.op.join:
+      values:
+        - - 1
+        - - 2
```

Comment:
And maybe even more nesting here, a otk.join inside another otk.join (probably a new testcase).

Reply:
I added this in the new `09-op` testcase as well.